### PR TITLE
Final updates for gallantries

### PR DESCRIPTION
**Pull request by hexylena**
Added some draft mode LPs (@nomadscientist we'll support draft learning pathways now) and a real quick n dirty asynchronous training handbook

fixes https://github.com/galaxyproject/training-material/issues/4419

### CODE REVIEW

File: learning-pathways/io3.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      - name: repeatmasker
+        topic: genome-annotation
+      - name: lncrna
+        topic: genome-annotation
+      - name: apollo-euk
+        topic: genome-annotation
+
+  - section: "Year 3: Official Gene Set"
+    description: |
+       ⚠️  **TODO** ⚠️One of the key tasks in annotation is producing an official gene set (OGS), and ensuring integrity and validation of all of the curated annotations. This will also further familiarise students with public databases and the process for submitting datasets. [SC3.1, SC3.5]
```
Comment (hexylena) [inline learning-pathways/io3.md:81]: @abretaud not sure if you have anything you already want to link to here, or not

File: learning-pathways/io5.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+pathway:
+
+  - section: "Hybrid Lesson Development Guide"
+    description: |
+      Covers an introduction to pedagogy for instructors and how to adapt training materials to hybrid methodology and key points and indicators that instructors should use when evaluating or adapting training materials to hybrid training delivery [SC5.1]
+    tutorials:
+      - name:
+        topic:
```
Comment (hexylena) [inline learning-pathways/io5.md:29]: @shiltemann will take care of the updates for this one (maybe as FAQs in the contributing tutorial)

File: learning-pathways/io3.md (hunk 1 above)
Comment (abretaud) [reply learning-pathways/io3.md:81]: I have a [pypi](https://pypi.org/project/gogstools/) and a [bioconda](https://anaconda.org/bioconda/gogstools) package now, I just need to create a galaxy tool, but I won't have time before november :/

**Review by abretaud — APPROVED** [review]

File: learning-pathways/io3.md (hunk 1 above)
Comment (hexylena) [reply learning-pathways/io3.md:81]: ok, I will make a draft mode material then just to have something to link to, and we'll identify it as delayed

**Review by shiltemann — APPROVED** [review]

**Pull request merged**